ui/app/pages/token/actionable-message/actionable-message.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+export default function ActionableMessage ({
+  shown = true,
+  message = '',
+  actions = [],
+  className = '',
+}) {
+  return (
+    shown
+      ? (
+        <div className={classnames('actionable-message', className)}>
+          <div className="actionable-message__message">
+            {message}
+          </div>
+          <div className="actionable-message__actions">
+            {
+              actions.map(({ label, onClick, actionClassName }, index) => (
+                <div
+                  className={classnames('actionable-message__action', actionClassName)}
+                  onClick={onClick}
+                  key={`actionable-message-action-${index}`}
+                >
+                  {label}
+                </div>
+              ))
+            }
+          </div>
+        </div>
+      )
+      : null
+  )
+}
+
+ActionableMessage.propTypes = {
+  shown: PropTypes.bool,
+  message: PropTypes.string.isRequired,
+  actions: PropTypes.shape({
```

Comment:
I believe this should be `PropTypes.arrayOf(PropTypes.shape(...))`

Reply:
Another Q: Are we giving too much flexibility in this API? Will we ever want *three* actions? I haven't seen the usage of it yet, or the designs, so I can't be sure... but if we will ever only show at most two actions then I think the API should be changed to 'primaryAction' and 'secondaryAction' or something to that effect and remove the array and iteration.

Reply:
Forgot to remove this proptype for actions 😬